sw/device/lib/runtime/hart.c

```diff
@@ -11,7 +11,7 @@
 extern void wait_for_interrupt(void);
 
 void busy_sleep_micros(size_t microseconds) {
```

Comment:
This is a strange name and data type for a function commonly called `usleep(unsigned int usec)`, so if you want to change the data type that's fine with me.

Reply:
I agree. I will update this PR.

Reply:
I believe I renamed it from `usleep` because that function tends to have connotations of yielding to a scheduler. I think a spin-sleep should be marked as such.

Also, as Sam said, you want a larger data type, though you should prefer to spell it with a stdint.h name, like `uint64_t`.